src/cargo/ops/tree/mod.rs

```diff
@@ -86,6 +86,32 @@ impl FromStr for Prefix {
     }
 }
 
+#[derive(Clone, Copy)]
+pub enum DisplayDepth {
+    MaxDisplayDepth(u32),
+    Workspace,
+}
```

Comment:
As this is dead code, please quash it into the commit that follows

It should also just be a newtype at this point as there won't be any workspace support yet

Reply:
Thanks for squashing.  This should also be switched to a new type, only parsing numbers.

Every commit should be atomic and this commit accepts new user input and then ignores it.

Reply:
The adding of workspace support should be in the commit that reads the variable

Reply:
Oh, I understood

Reply:
It is being read - though not used - here in the same commit 😅 
https://github.com/rust-lang/cargo/pull/14928/commits/a66e96a2b6bcd81d16c461108fc47d2f41127e6d#diff-52fd799ef4f821832efb7407f8d6d8501d8067e08dfb6f189203e19942c30d1bR409-R413